src/report.sh

```diff
@@ -429,9 +458,9 @@ function parse_report_options()
     esac
   done
 
-  if [[ -n "${options_values['STATISTICS']}" ]] && [[ -n "${options_values['OUTPUT']}" ]]; then
-    options_values['ERROR']='--output cannot be used with --statistics.'
-    return 22 # EINVAL
+  if [[ -z "${options_values['STATISTICS']}" && -z "${options_values['POMODORO']}" && -z "${options_values['ALL']}" ]]; then
```

Comment:
There is no `options_values['ALL']` you don't need to check it here.
```suggestion
  if [[ -z "${options_values['STATISTICS']}" && -z "${options_values['POMODORO']}" ]]; then
```